app/client/cypress/integration/national.spec.js

```diff
@@ -0,0 +1,188 @@
+    cy.get('span[data-term="Drinking Water Health-based Violations"]').click();
+    cy.get('#glossary')
+      .should('exist')
+      .and('be.visible');
+  });
+});
+
+describe('National Water Conditions tab', () => {
+  beforeEach(() => {
+    cy.visit('/national');
+  });
+
+  it('Clicking the “DISCLAIMER” button displays the disclaimer popup', () => {
+    const text = /^EPA has posted information through this/;
+
+    // verify opening the disclaimer
+    cy.findByText('Disclaimer').click();
+    cy.findByText(text).should('exist');
+
+    // verify closing the disclaimer
+    cy.findByTitle('Close disclaimer').click();
+    cy.findByText(text).should('not.exist');
+  });
+
+  it('From one of the sub-tabs, clicking “Expand All/Collapse All” expands/collapses the content', () => {
+    // verify that only "Expand All" is visible
+
+    const text = /^Biological condition tells us how healthy a waterbody is/;
+
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('not.exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('exist');
+
+    // Click "Expand All" and verify that "Collapse All" is now visible
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .click();
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('not.exist');
+
+    cy.findByText(text)
+      .filter(':visible')
+      .should('exist');
+
+    // Click "Collapse All" and verify that "Expand All" is now visible and accordion text is hidden
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .click();
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('not.exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('exist');
+    cy.findByText(text).should('not.exist');
+  });
+
+  it('Clicking the Nutrient Pollution link opens a new tab', () => {
+    // since Cypress doesn't support multiple tabs, we'll do the next best thing
+    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
+
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'href', 'https://www.epa.gov/nutrientpollution');
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'target', '_blank');
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'rel', 'noopener noreferrer');
```

Comment:
Here's where regex helps you. There aren't multiple cases on the page where "Nutrient Pollution" exists (with that capitalization), other than in the link. As always, prefer `dom-testing-library` selectors over cypress `get()` (notice a theme here 😉).

```suggestion
    cy.findByText(/Nutrient Pollution/)
      .should('have.attr', 'href', 'https://www.epa.gov/nutrientpollution')
      .should('have.attr', 'target', '_blank')
      .should('have.attr', 'rel', 'noopener noreferrer');
```